Xbim.IDS.Validator.Core/IdsModelValidator.cs

```diff
@@ -183,11 +183,11 @@ private Audit.Status ValidateIdsSchema(Xids idsSpec, ILogger userLogger)
                 // Note: won't support zipped IDS upgrades, JSON etc.
                 var targetVersion = IdsVersion.Ids1_0;
                 var currentVersion = idsSchemaMigrator.GetIdsVersion(idsFile);
-                logger.LogWarning("IDS schema {oldVersion} is out of date for {file}. Applying in-place upgrade to latest {version} schema.",
-                    currentVersion, idsFile, targetVersion);
+                logger.LogWarning("IDS schema {oldVersion} is out of date. Applying in-place upgrade to latest {version} schema.",
+                    currentVersion, targetVersion);
                 if (idsSchemaMigrator.MigrateToIdsSchemaVersion(idsFile, out var upgraded, targetVersion))
                 {
-                    logger.LogInformation("IDS file {file} upgraded in-place to latest schema", idsFile);
+                    logger.LogInformation("IDS file upgraded in-place to latest schema");
```

Comment:
@AmirEmilV Doesn't seem like a very helpful change! What happens when running with many IDS scripts. Assuming userlogger is user facing, and logger is developer-facing, what are we hiding this info for?